### ISSUE TITLE
ie8 only install, vboxheadless left running, 

### ISSUE THREAD

**Issue by frumbert** [xdissent/ievms#151]
i have virtualbox running with ie9 already which I installed about 18 months ago using this script. Since then it's evidently changed quite a bit. I'd tried to install IE8 and got so far:

Attaching ievms.iso
Starting VM IE8 - WinXP
Waiting for VM "IE8 - WinXP" to power on...
VM "IE8 - WinXP" has been successfully started.
Waiting for IE8 - WinXP to shutdown...
Ejecting ievms.iso
Starting VM IE8 - WinXP
Waiting for VM "IE8 - WinXP" to power on...
VM "IE8 - WinXP" has been successfully started.
Waiting for IE8 - WinXP to be available for guestcontrol...
VBoxManage: error: The guest execution service is not ready (yet).
Copying IE8 installer to Desktop

at which point the process stopped, and I notice in VirtualBox that the machine is running still. The tail of the log is

00:00:30.716 Executing guest process "vbox_stat" as user "IEUser" ...
00:00:30.783 PCNet#0: Init: ss32=1 GCRDRA=0x023d1420[64] GCTDRA=0x023d1020[64]
00:00:30.953 Guest process (PID 1380) started
00:00:30.958 Guest process (PID 1380) exited normally (exit code: 0)
00:00:30.961 Creating guest directory "/Documents and Settings/IEUser/Desktop" as  user "IEUser" ...
00:00:30.961 Executing guest process "vbox_mkdir" as user "IEUser" ...
00:00:31.116 Guest process (PID 1608) started
00:00:31.231 Guest process (PID 1608) exited normally (exit code: 0)
00:00:31.233 Copying file "/Users/tim/.ievms/IE8-WindowsXP-x86-ENU.exe" to guest "/Documents and Settings/IEUser/Desktop/Install IE8.exe" (16883056 bytes) ...
00:00:31.233 Executing guest process "vbox_cat" as user "IEUser" ...
00:00:31.338 Guest process (PID 1680) started

basically stalled at this point. A) can't stop the machine since it's being held/run by VboxHeadless, B) can't just open the machine and fix it myself since it's already running by a service, not interactively.
